lib/index.js

```diff
@@ -58,7 +58,8 @@ var epilogue = {
       pagination: args.pagination,
       updateMethod: this.updateMethod,
       search: args.search,
-      sort: args.sort
+      sort: args.sort,
+      reloadOnCreate: args.reloadOnCreate
```

Comment:
maybe it's a good idea to make this an array or an object? e.g.
```
reload: {
  onCreate: true,
  onUpdate: false
}
```
or
```
reload: ['create', 'update'];   // associated with controller
```

you might not want to ever reload the object (even on update)